assets/src/edit-story/components/panels/preset/stylePreset/panel.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { rgba } from 'polished';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import PresetPanel from '../presetPanel';
+import { Remove } from '../../../../icons';
+import generatePatternStyles from '../../../../utils/generatePatternStyles';
+import { BACKGROUND_TEXT_MODE } from '../../../../constants';
+import { useStory } from '../../../../app/story';
+import stripHTML from '../../../../utils/stripHTML';
+import { generatePresetStyle } from '../utils';
+
+const REMOVE_ICON_SIZE = 18;
+
+const Preset = styled.button`
+  background-color: ${({ theme }) => rgba(theme.colors.fg.white, 0.16)};
+  display: inline-block;
+  border-color: transparent;
+  position: relative;
+  cursor: pointer;
+  border-width: 0;
+  border-radius: 4px;
+  padding: 3px;
+  height: 100%;
+  width: 100%;
+  svg {
+    width: ${REMOVE_ICON_SIZE}px;
+    height: ${REMOVE_ICON_SIZE}px;
+    position: absolute;
+    top: calc(50% - ${REMOVE_ICON_SIZE / 2}px);
+    left: calc(50% - ${REMOVE_ICON_SIZE / 2}px);
+  }
+`;
+
+const HighLight = styled.span`
+  padding: 0 2px;
+  ${({ background }) => generatePatternStyles(background)}
+  box-decoration-break: clone;
+`;
+
+const LINE_HEIGHT = 20;
+const TextWrapper = styled.div`
```

Comment:
Shouldn't this be a button? I can tab to it, but I cannot click it using either <kbd>enter</kbd> or <kbd>space</kbd> (the former is partially due to #4239 not being merged yet).

Reply:
Hmm, it actually does work when using `Space`. But then, when clicking `Enter` and afterward `Space` again, the `Space` stops working as well. Will check.

(`Preset` that contains the `TextWrapper` is a `button`).

Reply:
Actually, it was related to #4239 -- added a [temporary fix](1468ff1), will remove once your PR gets merged.